docs/src/index.md

```diff
@@ -1,13 +1,25 @@
-# DataFrames.jl
+<img src="assets/logo.png" align="left" style="padding-right:10px"; width=20%"></img>
 
-Welcome to the DataFrames.jl documentation!
+## DataFrames.jl
 
-This resource aims to teach you everything you need to know to get up and
-running with tabular data manipulation using the DataFrames.jl package.
+Welcome to the `DataFrames.jl `documentation!
```

Comment:
why backquote `DataFrames.jl` here and below? This is not code. We consistently use package names in plain text in the docs/docstrings in DataFrames.jl.

Reply:
ok. If that's the convention, it should be like that. For me these ones here are just a guide to the eye. I will fix this.